librosa/core/notation.py

```diff
@@ -131,7 +131,7 @@
 
 NOTE_RE = re.compile(
     r"^(?P<note>[A-Ga-g])"
-    r"(?P<accidental>[#♯𝄪b!♭𝄫♮]*)"
+    r"(?P<accidental>[#♯𝄪b!♭𝄫♮n]*)"
```

Comment:
Should we add support for this in the key parser as well?  I know it would be weird to have something like `G♮:maj`, but I could imagine it coming up in some weird circumstances (eg parsing a key derived from a modulation).

At any rate, it doesn't seem obviously weirder than `G𝄫:maj`, unless I'm missing something.

Reply:
I'm happy to do this. The simplest change will treat `G♮:maj` as equal to `G:maj`, which may or may not be in keeping with the spirit of the issue raised by @lostanlen. Since this is a completely theoretical key signature, I think it's probably fine to treat `G♮:maj` as `G:maj`.

Reply:
Yeah, I tend to agree - it seems like the only consistent way to handle it.  Would "G♮" ever mean anything *other* than "G", given how we already parse unaccidented notes?

Reply:
I agree, especially because we treat `Gb#:maj` as `G:maj`. The natural sign has a theoretical function only once we've established a key or another context (e.g. a bar in which an accidented note has already appeared). The PR is about making use of this function of the natural sign. When the natural sign appears in a key name (i.e. before a context has been established), it doesn't really have a function in music theory, and therefore I think we're safe to treat it simply as an additive 0 to the key name.